RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: minor
+
+This release adds magnitude constraints to the
+:func:`~hypotheses.strategies.complex_numbers` strategy.
```

Comment:
I’d be inclined to be more explicit – list the new parameters we’ve added. (I see `allow_infinity` and `allow_nan` have also appeared silently.)

Possibly also note that we’ve changed the internal implementation, and why that might be significant?